hydro_cli/src/core/hydroflow_crate.rs

```diff
@@ -30,7 +30,7 @@ pub struct HydroflowCrate {
 
     pub built_binary: Option<String>,
     pub launched_host: Option<Arc<dyn LaunchedHost>>,
-    pub allocated_incoming: HashMap<String, ConnectionPipe>,
+    pub allocated_incoming: HashMap<String, (ConnectionPipe, Box<dyn Any + Send + Sync>)>,
```

Comment:
What's the `dyn Any` field for? Looks like its always ignored/seems like this is never inserted into?

Reply:
More comments thruout would be good